addon/components/cfb-form-editor/question/options.js

```diff
@@ -22,6 +30,8 @@ export default RenderComponent.extend({
   layout,
 
   intl: service(),
+  notification: service(),
+  apollo: service(),
```

Comment:
This needs to be the `queryManager` by `ember-apollo-client`: https://github.com/ember-graphql/ember-apollo-client/tree/v2.2.1#usage

Reply:
Changed